plugins/search/providers/elastic_search/batch/KExportMediaEsearchEngine.php

```diff
@@ -128,12 +131,22 @@ protected function retrieveEntryCategories ($entryId)
 		
 		$categoryEntryResult = KBatchBase::$kClient->categoryEntry->listAction($categoryEntryFilter, $pager);
 		
-		$result = array();
 		foreach ($categoryEntryResult->objects as $categoryEntry)
 		{
 			$result[] = $categoryEntry->categoryId;
 		}
 		
 		return $result;
 	}
-}
\ No newline at end of file
+	/**
+	 * @param int $timestamp
+	 * @param int $options
+	 * @return false|string
+	 */
+	protected function formatTimestamp($timestamp, $options) {
+		if($options != KalturaExportToCsvOptionsType::HUMAN_READABLE_DATES) {
```

Comment:
1. I suggets using switch case here, since we should assume that options will increasse, otherwise we can do it with much simpler method.